app/videobridge/client/views/videoFlexTab.js

```diff
@@ -59,7 +60,7 @@ Template.videoFlexTab.onRendered(function() {
 		const update = () => {
 			const { jitsiTimeout } = Rooms.findOne({ _id: rid }, { fields: { jitsiTimeout: 1 }, reactive: false });
 
-			if (jitsiTimeout && (new Date() - new Date(jitsiTimeout) + CONSTANTS.TIMEOUT < CONSTANTS.DEBOUNCE)) {
+			if (jitsiTimeout && (TimeSync.serverTime(), - new Date(jitsiTimeout) + CONSTANTS.TIMEOUT < CONSTANTS.DEBOUNCE)) {
```

Comment:
So if there's a bug somewhere in the interface that keeps triggering new Jitsi Video calls, we've now not fixed that issue but slowed it down?

I'm automatically creating a new Jitsi video call every morning apparently and I have no idea why. Unsure about the RocketChat version we're running though, will have to ask about that.